### PR TITLE
loudmouth 1.5.3

### PR DESCRIPTION
**Pull request by ilovezfs**
Also,
- add linked library dependency `gettext`
- run `make check`
- install the examples
- add a test

### CODE REVIEW

File: Formula/loudmouth.rb [hunk 1]
```diff
@@ -24,11 +23,25 @@ class Loudmouth < Formula
...
     system "make", "install"
+    (pkgshare/"examples").install Dir["examples/*.c"]
+  end
+
+  test do
+    cp pkgshare/"examples/lm-send-async.c", testpath
+    system ENV.cc, "lm-send-async.c", "-o", "test",
+      "-lloudmouth-1", "-lglib-2.0",
+      "-I#{include}/loudmouth-1.0",
+      "-I#{Formula["glib"].include}/glib-2.0",
```
Comment (DomT4) [inline Formula/loudmouth.rb]: Should possibly be `.opt_include` so we're consistent with a lot of other usage.

File: Formula/loudmouth.rb [hunk 2]
```diff
@@ -24,11 +23,25 @@ class Loudmouth < Formula
...
+      "-I#{Formula["glib"].lib}/glib-2.0/include"
```
Comment (DomT4) [inline Formula/loudmouth.rb]: (and `.opt_lib` here, as well, if the above is changed)

**Comment by DomT4** [comment]
Couple of nits but LGTM otherwise.

**Comment by ilovezfs** [comment]
What's the logic for the `opt_*` in `test do`? I'm interested if it breaks against the current version not the randomly installed possibly outdated version. During CI it seems they'd be identical. On a local machine, non-opt would avoid false negatives/positives caused by testing against a non-current version. Also, since it's a direct dependency in this case, it seems the current version would have to be the same as the opt version, but maybe I'm missing something that's good about opt here.

**Comment by DomT4** [comment]
In testing it's not actually massively important, it's really more about consistency there than much else. `brew test xyz` is incredibly rarely ran (GA backs this up), and anything submitted as a PR will be tested by CI without that user configuration issue, so it's not something we need to worry about particularly.

**Pull request closed without merging**